starter-code/memory.js

```diff
@@ -1,11 +1,227 @@
+  		{ name: "the avengers",    img: "./img/the-avengers.jpg" },
+  		{ name: "thor",            img: "./img/thor.jpg" },
+  	];
+  this.picked_cards  = [];
+  this.selectedPairs_clicked = 0;
+  this.selectedPairs_guessed = 0;
+  this._shuffleCard();
+};
+
+// this function just takes the array of cards above and shuffles them into a random order
+MemoryGame.prototype._shuffleCard = function() {
+  var counter = this.Cards.length;
+
+  while (counter > 0) {
+    index = Math.floor(Math.random() * counter);
+    counter--;
+    temp = this.Cards[counter];
+    this.Cards[counter] = this.Cards[index];
+    this.Cards[index] = temp;
+  }
+  return;
+};
+
+//---------------------------------------------------------------------------
+//a cheap way to access/initialize variables
+
+var idx = 0; //setting an index to use throughout the function
+var countChocula = 0; //he's really good at counting
+var imgTag = "";
+var clickedSrc = "";
+var currentDiv = "";
+var lastDiv = [];
+var cardsclicked = 0;
+var cardsRemain = 0;
+var selectedPairs = [];
+var imgIDs = [];
+var idNum = 0;
+
+//-------------------------------------------------------------------------------
+//this function lets me create some logic for the cards that don't match
+//first thing it does is log...so that you know the card is about to be cleared
+
+var clearCards = function() {
+    console.log("CARD WILL BE CLEARED");
+    $(".col").toggleClass("blocked"); //then it blockes the user from clicking anything until timeout.
+    setTimeout(function(){ //hides the img after 1 second, also removes the blocked class
+      imgTag.toggleClass(); //runs toggle and adds hidden class after three seconds of being clicked
+      $(currentDiv).toggleClass('hidebg'); //selects the currenet div clicked and adds/toggles the img visibility
+      // $(currentDiv).removeClass("clicked"); //for ever item that is defined as clicked, removes this class
+      $(".col").toggleClass("blocked"); //removes the click block
+  },1000); //closes the timeout function
+};
+
+
 
 //******************************************************************
 // HTML/CSS Interactions
 //******************************************************************
+var memoryGame;
+$(document).ready(function() {
+  memoryGame = new MemoryGame();
+//-------------------------------------------------------------------------------
+
+  $('.col').each(function(){ //an each loop that will iterate through each box/column
+    var currentImg = $(this).children("img"); //selects the img tag in each box/column
+    $(currentImg).prop('src', memoryGame.Cards[idx].img); //targets the src property of the img tag selected and replaces it with Card array img src
+    idx+=1; //increases the index value so that you can continue iterating through the loop
+
+  }); //close the each loop
 
+//-------------------------------------------------------------------------------
+//-------------------------------------------------------------------------------
```

Comment:
When I click a card, the 1st card doesn't stay displayed while I look for the second one.